modules/articles/client/views/view-article.client.view.html

```diff
@@ -3,12 +3,12 @@
     <h1 ng-bind="article.title"></h1>
```

Comment:
We don't seem to be using this file - you probably changed this while you were testing the article functionality? But we can probably get rid of these changes just to clean up the pull request